<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1422</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MotorWidget" name="motor_widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>343</width>
      <height>396</height>
     </rect>
    </property>
   </widget>
   <widget class="MotorWidget" name="motor_widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>90</y>
      <width>343</width>
      <height>396</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Delay Lines&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="MotorWidget" name="motor_widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>90</y>
      <width>343</width>
      <height>396</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="dl_dl1_label_opcua_status">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="MotorWidget" name="motor_widget_1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>343</width>
      <height>396</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1422</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_dlPositions">
    <property name="title">
     <string>DL Positions</string>
    </property>
    <addaction name="actionSave_Current_Positions"/>
    <addaction name="actionRecall_Positions"/>
   </widget>
   <addaction name="menu_dlPositions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_Current_Positions">
   <property name="text">
    <string>Save Current Positions</string>
   </property>
  </action>
  <action name="actionRecall_Positions">
   <property name="text">
    <string>Recall Positions</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MotorWidget</class>
   <extends>QWidget</extends>
   <header>motorwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
